frontend/resourceadm/pages/ResourcePage/ResourcePage.tsx

```diff
@@ -239,15 +197,45 @@ export const ResourcePage = (): React.JSX.Element => {
   return (
     <div className={classes.resourceWrapper}>
       <div className={classes.leftNavWrapper}>
-        <LeftNavigationBar
-          upperTab='backButton'
-          tabs={getTabs()}
-          backLink={getResourceDashboardURL(org, app)}
-          backLinkText={t('resourceadm.left_nav_bar_back')}
-          selectedTab={
+        <StudioContentMenu.Static
+          onChangeTab={(tabId: NavigationBarPage) => {
+            if (tabId !== 'back') {
+              navigateToPage(tabId);
+            }
+          }}
+          selectedTabId={
             currentPage === migrationPageId && !isMigrateEnabled() ? aboutPageId : currentPage
           }
-        />
+        >
+          <StudioContentMenu.LinkTab
+            tabId={'back'}
```

Comment:
Maybe we can do like this for all? 😄 
```suggestion
            tabId='back'
```